packages/mobile/.env

```diff
@@ -1,8 +1,10 @@
 ENVIRONMENT=local
 DEFAULT_TESTNET=integration
-# -1 == ZeroSync, 5 == Ultralight, see src/geth/consts.ts for more info
+# If ZERO_SYNC_ENABLED_INITIALLY, local geth will not run initially. 
+# If toggled on, it will use DEFAULT_SYNC_MODE. See src/geth/consts.ts for more info
+ZERO_SYNC_ENABLED_INITIALLY=true
```

Comment:
Thanks for clarifying the this. My next question is then: should this be true? My understanding from my chat with Connor about this a while back was that we'd keep using geth by default during development, so we could catch bugs early (since prod still uses geth)

Reply:
Good point for usually using geth during development. I'll have this be false for all env files for now.